examples/fluids/setup-boundary.h

```diff
@@ -55,24 +55,31 @@
 //
 // *****************************************************************************
 CEED_QFUNCTION(SetupBoundary)(void *ctx, CeedInt Q,
-                      const CeedScalar *const *in, CeedScalar *const *out) {
+                              const CeedScalar *const *in, CeedScalar *const *out) {
   // Inputs
-  const CeedScalar (*J)[3][CEED_Q_VLA] = (const CeedScalar(*)[3][CEED_Q_VLA])in[0],
-                   (*w) = in[1];
+  const CeedScalar (*J)[3][CEED_Q_VLA] = (const CeedScalar(
+      *)[3][CEED_Q_VLA])in[0],
+                                         (*w) = in[1];
   // Outputs
   CeedScalar (*qdataSur)[CEED_Q_VLA] = (CeedScalar(*)[CEED_Q_VLA])out[0];
 
   CeedPragmaSIMD
   // Quadrature Point Loop
   for (CeedInt i=0; i<Q; i++) {
     // Setup
-    const CeedScalar dxdX[3][2] = {{J[0][0][i],
-                                    J[1][0][i]},
-                                   {J[0][1][i],
-                                    J[1][1][i]},
-                                   {J[0][2][i],
-                                    J[1][2][i]}
-                                   };
+    const CeedScalar dxdX[3][2] = {{
+        J[0][0][i],
+        J[1][0][i]
+      },
+      {
+        J[0][1][i],
+        J[1][1][i]
+      },
+      {
+        J[0][2][i],
+        J[1][2][i]
+      }
+    };
```

Comment:
Does this seem fine?

Reply:
That sort of stupidness from `make style` can be prevented by using `// *INDENT-OFF*` and `// *INDENT-ON*`.

Reply:
Ah, okay. Thanks.

Reply:
Thanks @LeilaGhaffari  for fixing that!